package.json

```diff
@@ -13,7 +13,7 @@
 	"type": "module",
 	"exports": {
 		"types": "./index.d.ts",
```

Comment:
Keep `index.d.ts` at the top-level because some TypeScript consumers seem to assume that (instead of reading `export.types`, and I've encountered issues there before... But I might be superstitious about this, and it might just work when under `source/`, not sure... :thinking:

Reply:
It's only a problem if their config is not properly configured to ESM, meaning not having `"module": "node16", "moduleResolution": "node16"`. Having it at the top-level lets them circumvent this, which means it could potentially cause other problems, and we have to deal with support for that too. So I would prefer to keep it in the `source` directory.

Reply:
Moved it to `source`. :+1: